README.md

```diff
@@ -25,6 +25,17 @@ Also has a new minor fixes and quality of life improvements like:
 
 - `new.reddit.com`
 
+
+## Contributing
+
+Install `web-ext` by running `npm install`. Then run `make run` which will
+trigger `web-ext` to start a live-reloading development server and launch the
+extension in a browser window for testing.
```

Comment:
````suggestion
```
make run
```

This will trigger `web-ext` to start a live-reloading development server and launch the extension in a browser window for testing.
````